### PR TITLE
zh-tutorial-translation

### PR DESCRIPTION
**Pull request by kingswanwho**
# [DOC UPDATE] Translate Tutorial to Chinese 13/15

## Description

Translate a tutorial to Chinese:

040-lesson-2-run-queries-with-ansi-sql.md

Overall 13 tutorials have been translated, 2 tutorials to go.

## Documentation

Please refer to Chinese tutorial.

## Testing

This is a doc update, no test here.

### CODE REVIEW

**Review by luocooong — COMMENTED** [review]
@kingswanwho Thanks for the contribution, please note the comments.

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md [hunk 1]
```diff
@@ -6,28 +6,21 @@ lang: "zh"
 ---
 ## Goal
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md]: resolved

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md [hunk 2]
```diff
@@ -6,28 +6,21 @@ lang: "zh"
...
+* 联接 Hive、MapR-DB 和文件系统数据源。
+* 使用表和列的别名。
+* 创建 Drill 视图。
 
-## Aggregation
+## 聚合
 
 
-### Set the schema to hive:
+### 设定 hive 的 schema
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md]: resolved

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md [hunk 3]
```diff
@@ -94,14 +84,13 @@ is a reserved word in SQL.
...
+请注意 SUM 函数结果的别名。Drill 支持列别名和表别名。
 
-## HAVING Clause
+## HAVING 子句
 
-This query uses the HAVING clause to constrain an aggregate result.
+此查询使用 HAVING 子句来约束聚合结果。
 
-### Set the workspace to dfs.clicks
+### 将工作区设置为 dfs.clicks
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md]: resolved

File: _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md [hunk 4]
```diff
@@ -313,13 +284,13 @@ supports the creation of metadata in the file system.
     |----------|-------------------|-----------|----------|--------|----------|-------------|
     1 row selected
 
-Once the users know what data is available by exploring it directly from the file system, views can be used as a way to read the data into downstream tools such as Tableau and MicroStrategy for analysis and visualization. For these tools, a view appears simply as a “table” with selectable “columns” in it.
+用户可以直接从文件系统中探索来了解哪些数据可用，视图可以将数据读入下游工具进行分析和可视化，就像 Tableau 和 MicroStrategy 一样。对于这些工具，视图仅显示为“表格”，其中包含可选择的“列”。
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/learn-drill-with-the-mapr-sandbox/040-lesson-2-run-queries-with-ansi-sql.md]: resolved

**Review by luocooong — APPROVED** [review]
LGTM. +1

**Pull request merged**